libcontainer/apparmor/apparmor.go

```diff
@@ -24,16 +21,34 @@ func IsEnabled() bool {
 	return false
 }
 
+func setprocattr(tid int, attr, value string) error {
+	path := fmt.Sprintf("/proc/%d/attr/%s", tid, attr)
```

Comment:
While this works, I think this should be `/proc/self/attr/%s`. Under AppArmor you can only change your own `attr`, so I think `/proc/self` makes more sense than providing a `tid` argument.

Reply:
Updated, PTAL.